<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWakeJobDialog</class>
 <widget class="QDialog" name="EditWakeJobDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wake Job Configuration</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Job name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="nameEdit">
       <property name="text">
        <string>Wake</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Max retries</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="retriesBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Timeout for each retry</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="timeoutBox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
